administration/src/cards/extensions/EMailExtension.ts

```diff
@@ -0,0 +1,24 @@
+import { Extension } from './extensions'
+
+class EMailExtension extends Extension<string, null> {
+  public readonly name = EMailExtension.name
+
+  setInitialState() {}
+  createForm() {
+    return null
+  }
+  causesInfiniteLifetime() {
+    return false
+  }
+  fromString(state: string): void {
+    this.state = state
+  }
+  toString(): string {
+    return this.state ?? ''
+  }
+  isValid() {
```

Comment:
not sure if we need another check. But this field is optional, so if a card will be created without mail address (not created from application) it should not block the card creation